awx/ui_next/src/api/models/WorkflowJobTemplates.js

```diff
@@ -49,7 +49,21 @@ class WorkflowJobTemplates extends SchedulesMixin(NotificationsMixin(Base)) {
   }
 
   readAccessList(id, params) {
-    return this.http.get(`${this.baseUrl}${id}/access_list/`, { params });
+    return this.http.get(`${this.baseUrl}${id}/access_list/`, {
+      params,
+    });
+  }
+
+  readSurvey(id) {
+    return this.http.get(`${this.baseUrl}${id}/survey_spec/`);
+  }
+
+  updateSurvey(id, survey = null) {
```

Comment:
would this not be survey = {}?

Reply:
I'm not sure we need any default value here... seems like it's an essential parameter

Reply:
we don't need that default value at all.  Good catch.  I'll remove it from here and from `JobTemplates.js`